MonstieWash/Assets/Scripts/Erasing/Eraser.cs

```diff
@@ -318,7 +325,7 @@ private Vector2Int GetPixelCoordinatesOnTexture(Texture2D texture, int pixel)
     /// <returns>Whether a change was made to the erasable mask.</returns>
     private bool ApplyPixels(byte toolMaskPixelAlpha, byte[] erasableMaskPixels, int[] drawingPixels)
     {
-        var pixelStrength = toolMaskPixelAlpha * (tool.Strength / 100f);
+        var pixelStrength = toolMaskPixelAlpha * (tool.Strength * Time.deltaTime / 100f);
```

Comment:
This is an attempt to make it framerate independent. Tbh haven't tested it in depth but I think it does the trick. Dampening the effectiveness when at a high framerate and boosting it when at low framerate since erasing is done every frame.